source-container-build/app/source_build.py

```diff
@@ -194,9 +194,9 @@ def parse_cli_args():
         "skip handling sources of base image.",
     )
     parser.add_argument(
-        "--cachi2-artifacts-dir",
+        "--prefetch-artifacts-dir",
```

Comment:
- Please add `--prefetch-artifacts-dir` as a new argument.
- Make `--cachi2-artifacts-dir` and `--prefetch-artifacts-dir` exclusive.
- For backward-compatibility, make `--cachi2-artifacts-dir` still work, but logging deprecation message for it.